tests/ert3/console/integration/test_cli.py

```diff
@@ -261,7 +317,15 @@ def test_cli_export_polynomial_evaluation(tmpdir):
     with unittest.mock.patch.object(sys, "argv", args):
         ert3.console.main()
 
-    args = ["ert3", "run", "evaluation"]
+    args = [
+        "ert3",
+        "run",
+        "evaluation",
+        "some_file",
```

Comment:
The idea here was that `evaluation` is indeed the experiment and that it should hence not be necessary to specify any additional files...

Reply:
So where would you specify the configuration? is it a part of the `init` phase, and should we persist the initialization at that stage?

Reply:
Well, for now a `workspace` expects there to be a `parameters.yml` and an `environments.yml` in the root folder. Every experiment is expected to contain both an `experiment.yml` and an `ensemble.yml`. So, basically a single experiment is configured as the union of multiple files with expected names within a folder.

What is configured in the experiment and in the workspace will probably have to change and I suspect that some of them should be possible to configure both places. However, the intent is to separate data configuration, modelling configuration and approach configuration so that one can reuse the same forward model in multiple experiments on different data, reuse the same data as input to different forward models, use the same data and forward model in different experiments and so forth.

However, I think we need to get some learning on how this work process is by getting implemented at least one other experiment type then `evaluation` to really get started figuring out where these boundaries should be 🤷